src/airflow_declarative/trafaret.py

```diff
@@ -193,6 +194,25 @@ def cast_interval(value):
     return value
 
 
+def cast_crontab_or_interval(value):
+    try:
+        return cast_interval(value)
+    except t.DataError:
+        if not isinstance(value, str):
+            raise
+        # This is definitely not a valid time interval.
+        # Perhaps this is a cron expression?
+        try:
+            # Airflow uses `croniter` as well.
+            croniter(value)
+        except ValueError as e:
+            raise t.DataError(
+                "Invalid timedelta or crontab expression: %s" % str(e)
```

Comment:
Technically, this will leave only latest reason of fail, in over wordes - croniter issue. Could be better if we can have both descriptions here, but since we done pretty good exception out from `cast_interval` that's fine case for future improvement.